milli/src/error.rs

```diff
@@ -12,6 +12,10 @@ use crate::{DocumentId, FieldId};
 
 pub type Object = Map<String, Value>;
 
+pub fn is_reserved_keyword(keyword: &str) -> bool {
+    ["_geo", "_geoDistance"].contains(&keyword)
```

Comment:
`_geoPoint` seems to leak here; you defined it in the pest grammar.

Reply:
Oops, yes, effectively. And `_geoRadius` was also handled in the pest grammar as a bad usage, so it's pretty logical to put it here too.

Thank you, I added this in this [commit](https://github.com/meilisearch/milli/pull/322/commits/f4b8e5675d903161b262910126cbf6ebf461f89a) with a bunch of tests :smile: